pkg/trace/obfuscate/sql.go

```diff
@@ -290,8 +287,18 @@ func attemptObfuscation(tokenizer *SQLTokenizer) (*ObfuscatedQuery, error) {
 		if token == LexError {
 			return nil, fmt.Errorf("%v", tokenizer.Err())
 		}
-		for _, f := range filters {
-			if token, buff, err = f.Filter(token, lastToken, buff); err != nil {
+
+		if token, buff, err = discard.Filter(token, lastToken, buff); err != nil {
+			return nil, err
+		}
+		if token, buff, err = replace.Filter(token, lastToken, buff); err != nil {
+			return nil, err
+		}
+		if token, buff, err = grouping.Filter(token, lastToken, buff); err != nil {
+			return nil, err
+		}
+		if filterTableFinder {
```

Comment:
```suggestion
		if config.HasFeature("table_names") {
```
I guess we can avoid that variable.

Reply:
Good point. I think that's a sign that we should make `config.HasFeature` more efficient. Perhaps by loading the features only once at load time.